eng/pipelines/coreclr/templates/build-job.yml

```diff
@@ -84,16 +84,9 @@ jobs:
         value: true
     - name: officialBuildIdArg
       value: ''
-    - name: ibcOptimizeArg
```

Comment:
You also need to change/remove the condition for restoring internal tools to run on Linux as well no? It is conditioned to only restore in Windows for CoreCLR jobs.